starter-code/src/clue.js

```diff
@@ -1,94 +1,208 @@
+  last_name: "Mustard",
+  color: "yellow",
+  description:
+    "He is a former football player who tries to get by on his former glory",
+  age: 62,
+  image: "https://metrouk2.files.wordpress.com/2016/07/colonel-mustard.jpg",
+  occupation: "Retired Football player"
+};
 
 // Weapons
 
-name: rope        ---   weight: 10
-name: knife       ---   weight: 8
-name: candlestick ---   weight: 2
-name: dumbbell    ---   weight: 30
-name: poison      ---   weight: 2
-name: axe         ---   weight: 15
-name: bat         ---   weight: 13
-name: trophy      ---   weight: 25
-name: pistol      ---   weight: 20
+var rope = {
+  name: "rope",
+  weight: 10
+};
+
+var knife = {
+  name: "knife",
+  weight: 8
+};
+
+var candlestick = {
+  name: "candlestick",
+  weight: 2
+};
+
+var dumbbell = {
+  name: "dumbbell",
+  weight: 30
+};
+
+var poison = {
+  name: "poison",
+  weight: 2
+};
+
+var axe = {
+  name: "axe",
+  weight: 15
+};
+
+var bat = {
+  name: "bat",
+  weight: 13
+};
+
+var trophy = {
+  name: "trophy",
+  weight: 25
+};
+
+var pistol = {
+  name: "pistol",
+  weight: 20
+};
 
 // Rooms
-name: Dinning Room
-name: Conservatory
-name: Kitchen
-name: Study
-name: Library
-name: Billiard Room
-name: Lounge
-name: Ballroom
-name: Hall
-name: Spa
-name: Living Room
-name: Observatory
-name: Theater
-name: Guest House
-name: Patio
-*/
+var DinningRoom = { name: "Dinning Room" };
+var Conservatory = { name: "Conservatory" };
+var Kitchen = { name: "Kitchen" };
+var Study = { name: "Study" };
+var Library = { name: "Library" };
+var BilliardRoom = { name: "Billiard Room" };
+var Lounge = { name: "Lounge" };
+var Ballroom = { name: "Ballroom" };
+var Hall = { name: "Hall" };
+var Spa = { name: "Spa" };
+var LivingRoom = { name: "Living Room" };
+var Observatory = { name: "Observatory" };
+var Theater = { name: "Theater" };
+var GuestHouse = { name: "Guest House" };
+var Patio = { name: "Patio" };
 
 // Characters Collection
-var charactersArray = [];
+var charactersArray = [
+  mrGreen,
+  drOrchid,
+  profPlum,
+  missScarlet,
+  mrsPeacock,
+  mrMustard
+];
 
 // Rooms' Collection
-var roomsArray = [];
+var roomsArray = [
+  DinningRoom,
+  Conservatory,
+  Kitchen,
+  Study,
+  Library,
+  BilliardRoom,
+  Lounge,
+  Ballroom,
+  Hall,
+  Spa,
+  LivingRoom,
+  Observatory,
+  Theater,
+  GuestHouse,
+  Patio
+];
 
 // Weapons Collection
-var weaponsArray = [];
+var weaponsArray = [
+  rope,
+  knife,
+  candlestick,
+  dumbbell,
+  poison,
+  axe,
+  bat,
+  trophy,
+  pistol
+];
+
+function randomSelector(array) {
+  var random = Math.floor(Math.random() * array.length);
+
+  //console.log(random);
+  //console.log(array[random]);
+
+  return array[random];
+}
+
+function pickMistery() {
+  var threeCards = [];
+  var card1 = randomSelector(charactersArray);
+  // console.log(card1.first_name);
+
+  var card2 = randomSelector(weaponsArray);
+  var card3 = randomSelector(roomsArray);
+
+  threeCards.push(card1.first_name + " " + card1.last_name);
```

Comment:
What you have done is correct, but it doesn't pass the test because it wants you to return a killer etc. By this it means it wants the full object of the killer, weapon and room
```
threeCards.push(card1, card2, card3) 
```
by pushing the full object it passes the test and then you can modify the last part to get it to pass the final test